browser/brave_wallet/external_wallets_importer_unittest.cc

```diff
@@ -141,17 +126,12 @@ class ExternalWalletsImporterUnitTest : public testing::Test {
   void SimulateIsExternalWalletInitialized(const std::string& json_str,
                                            bool* out_initialized) {
     ASSERT_NE(out_initialized, nullptr);
-    ExternalWalletsImporter cw_importer(
-        mojom::ExternalWalletType::CryptoWallets, browser_context());
     ExternalWalletsImporter mm_importer(mojom::ExternalWalletType::MetaMask,
                                         browser_context());
     auto json = base::test::ParseJsonDict(json_str);
-    cw_importer.SetStorageDataForTesting(json.Clone());
     mm_importer.SetStorageDataForTesting(json.Clone());
-    cw_importer.SetExternalWalletInstalledForTesting(true);
     mm_importer.SetExternalWalletInstalledForTesting(true);
-    *out_initialized = cw_importer.IsExternalWalletInitialized();
-    EXPECT_EQ(mm_importer.IsExternalWalletInitialized(), *out_initialized);
+    *out_initialized = mm_importer.IsExternalWalletInitialized();
   }
 
```

Comment:
`    EXPECT_EQ(mm_importer.IsExternalWalletInitialized(), *out_initialized);`
seems inadvertently removed

Reply:
This is intentional and I believe correct as I have it.  Before it was checking that the value is the same as the cw_importer set value, but we no longer have a cw_importer.